csharp/tools/tracing-config.lua

```diff
@@ -1,10 +1,28 @@
 function RegisterExtractorPack(id)
```

Comment:
Another approach would be running the database creation without any command, so running the autobuilder, but with a new extractor option specifying the binlog path. And then the autobuilder could start the extractor with the given path.